app/uk/gov/hmrc/apiplatform/modules/mfa/controllers/profile/MfaController.scala

```diff
@@ -50,40 +53,59 @@ class MfaController @Inject()(
                                val cookieSigner: CookieSigner,
                                val securityPreferencesView: SecurityPreferencesView,
                                authAppStartView: AuthAppStartView,
-                               accessCodeView: AccessCodeView,
+                               authAppAccessCodeView: AuthAppAccessCodeView,
                                qrCodeView: QrCodeView,
                                authAppSetupCompletedView: AuthAppSetupCompletedView,
-                               nameChangeView: NameChangeView
+                               nameChangeView: NameChangeView,
+                               mobileNumberView: MobileNumberView,
+                               smsAccessCodeView: SmsAccessCodeView,
+                               smsSetupCompletedView: SmsSetupCompletedView,
+                               selectMfaView: SelectMfaView
 
 )(implicit val ec: ExecutionContext,
   val appConfig: ApplicationConfig) extends LoggedInController(mcc) with WithUnsafeDefaultFormBinding {
 
+  private val scale = 4
+  val qrCode = QRCode(scale)
 
   def securityPreferences: Action[AnyContent] = atLeastPartLoggedInEnablingMfaAction { implicit request =>
+    val errorMessage = "Unable to obtain User information"
     thirdPartyDeveloperConnector.fetchDeveloper(request.userId).map {
       case Some(developer: Developer) => Ok(securityPreferencesView(developer.mfaDetails.filter(_.verified)))
-      case None => InternalServerError("unable to obtain User information")
+      case None => InternalServerError(errorHandler.standardErrorTemplate(errorMessage, errorMessage, errorMessage))
     }
   }
 
+  def selectMfaPage: Action[AnyContent] = atLeastPartLoggedInEnablingMfaAction { implicit request =>
+    Future.successful(Ok(selectMfaView(SelectMfaForm.form)))
+  }
+
+  def selectMfaAction: Action[AnyContent] = atLeastPartLoggedInEnablingMfaAction { implicit request =>
+    SelectMfaForm.form.bindFromRequest.fold(
+      form => Future.successful(BadRequest(selectMfaView(form))),
+      form => {
+        MfaType.withNameInsensitive(form.mfaType) match {
+          case SMS => Future.successful(Redirect(uk.gov.hmrc.apiplatform.modules.mfa.controllers.profile.routes.MfaController.setupSms()))
```

Comment:
Could `uk.gov.hmrc.apiplatform.modules.mfa.controllers.profile.routes._` be imported to shorten 4 lines in this file?

Reply:
Addressed